Formula/s/solana.rb

```diff
@@ -38,6 +38,9 @@ class Solana < Formula
   depends_on "protobuf" => :build
   depends_on "rust" => :build
 
+  depends_on "openssl@3"
+
+  uses_from_macos "llvm" => :build # for libclang
```

Comment:
Those are some pretty heavy dependencies to pull in. Is this a very popular feature that you're proposing here?

Reply:
I believe so, if you are developing against Solana the [docs](https://docs.solanalabs.com/cli/examples/test-validator) and [local dev guide](https://solana.com/developers/guides/getstarted/setup-local-development) both use the test validator and state it comes with the CLI, the install instructions for getting the CLI [call out installing via brew](https://docs.solanalabs.com/cli/install#use-homebrew) which I found confusing as it didn't come with the test validator as expected.

Reply:
Sorry for the bump @bevanjkay / @SMillerDev - would it be possible to get an approval or further the discussion on this please, I'm going to become blocked on this and may need to create my own tap'ed version to unblock myself.